src/Traits/Auditable.php

```diff
@@ -159,7 +159,8 @@ private function createAuditableChangesList(EventType $event_type, Model $model)
             case EventType::MODEL_UPDATED:
             case EventType::RELATION_UPDATED:
                 foreach ($modified as $attribute => $value) {
-                    if ($this->shouldAuditChange($attribute, $original[$attribute], $value)) {
+                    if ($this->shouldAuditChange($attribute, $original[$attribute], $value) &&
+                        !in_array($attribute, config('auditable.excluded_attributes') ?? [])) {
```

Comment:
`config()`'s second argument allows you to set a default when the config option is not defined: `config('auditable.excluded_attributes', [])` would do the trick instead of `?? []` (I think it would make it more readable that way).